share/commands/agent-check

```diff
@@ -7,4 +7,32 @@
 # @man - clean lock file if it is too big
 # @man - check and restore init files 
 # @man - check that promises have been properly copied
+# @man +
+# @man *Options*:
+# @man +
+# @man *-q*: run the agent in quiet mode (display only error messages)
+# @man +
+# @man *-c*: run the agent without color output
+
+. "${BASEDIR}/../lib/common.sh"
+
+QUIET=false
```

Comment:
Even if the option is named "quiet" to denote the default value of "not quiet", always use positive variables; ie VERBOSE.
Otherwise you end up using negation everywhere (if not quiet) and this makes the code unnecessarily hard to read.

So $QUIET -> $VERBOSE